styles/globals.css

```
@@ -625,6 +625,29 @@ video {
   animation: appear 0.3s ease-out;
 }
 
+.animate-disappear {
+  animation: appear 0.3s ease-in reverse forwards;
+}
+
+@keyframes slideUp {
```

Stylelint (17.2.0)

[error] Expected keyframe name "slideUp" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)